File: dayFour/java/afterBreak/TaskFive.java
import java.util.Scanner;

public class TaskFive{

    public static void main(String[] args){
    
        Scanner input = new Scanner(System.in);
        double total = 0.0;
        
        for(int count = 1; count <= 10; count++){
        
            System.out.println("Enter a score: ");
            double number = input.nextInt();
            if(number % 2 == 0){
                total += number;
            }
        }
        
        System.out.println("The sum of the even scores is "+total);
    }
}

File: dayFour/javaScript/taskFive.js
let total = 0;

for(let count = 1; count <= 10; count++){
    
    const scanner = require("prompt-sync")();
    let number = Number(scanner ("Enter score: "));
    if(number % 2 === 0){
        total += number;
        
    }
    
}

console.log("The sum of the even scores is "+total);
